tests/test_data_types.py

```diff
@@ -96,39 +115,56 @@ def test_audio_captions():
     assert wandb.Audio.captions([wbaudio5, wbaudio6]) == ['', caption2]
 
 
-def test_audio_transform():
-    audio = np.random.uniform(-1, 1, 44100)
+def test_audio_to_json():
+    audio = np.zeros(44100)
     with CliRunner().isolated_filesystem():
-        meta = wandb.Audio.transform(
-            [wandb.Audio(audio, sample_rate=44100)], ".", "test", 0)
-        assert meta == {'_type': 'audio',
-                        'count': 1, 'sampleRates': [44100], 'durations': [1.0]}
-        assert os.path.exists("media/audio/test_0_0.wav")
+        run = wandb.wandb_run.Run()
+        meta = wandb.Audio.seq_to_json(
+            [wandb.Audio(audio, sample_rate=44100)], run, "test", 0)
+        assert os.path.exists(os.path.join(run.dir, meta['audio'][0]['path']))
+        del meta['audio'][0]['run']
```

Comment:
i get why the `del` pattern is here, but it seems worth doing the upfront work to make these tests cleaner. either make a helper function thats like `assertDictKeysEqual` to that just checks equality for a partial list of keys, or (preferably) test how all the keys in the dict are set

Reply:
Yeah the del thing is a garbage hack. I'll try to improve it but I also don't really want to spend another week on this. It's time to get it in.

Reply:
So I got rid of the del's with a helper function, but now we don't get pytest's pretty printing of diffs in some cases. Not sure it's a net win. Maybe with more work we can get the pretty printing back. Pretty sure that's out of the scope of this change though.